modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/dev/public/js/static-env-utils.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+const Liferay = {
+	Language: {
+		get: (key) => {
```

Comment:
If you want you can use [shorter method definitions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Method_definitions)  (non blocking but I thought you'd be interested)

```js
const Liferay = {
    Language: {
      get(key) {
          // Do your thing
      }
   }
}
```

Reply:
It's the same to me removing an arrow and a colon. Instead what do you think about our whole Liferay utils contained in our `static-env-utils` file? They comes from `commerce-frontend-js`, we were using it for lot of modules; i guess they are replacing some LR utils exposed in the portal's frontend right? They are fine or should be aligned/removed/replaced? ty